Fix forge.config.js as per nathanlesage's review

cskeogh opened the pull request:
Fix forge.config.js as per:
- [x] [r637518977](https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637518977)
- [x] [r637519242](https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637519242)
- [x] [r637519809](https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637519809)
- [x] [r637519957](https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637519957)

Closes #64

cskeogh committed:
Add detail to `getLibraryPath` error message

Closes r637518977

cskeogh committed:
Add comments to getLibraryPath()

Addresses review comment
https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637519242

cskeogh committed:
Improve logic order in `postPackage`

Addresses review comment
https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637519809

cskeogh committed:
Correct let -> const, add comment in `postPackage`

Addresses review comment
https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637519957

ratherto commented on forge.config.js:
```diff
@@ -196,6 +186,12 @@ module.exports = {
             console.log(`Unable to bundle 'libappindicator3' (${err}).`)
           }
         }
+      } else if (isLinux && targetArch !== process.arch) {
```
else if (isLinux) would be enough

cskeogh replied:
Thank you. Incoproated.

cskeogh committed:
Remove unnecessary check targetArch

Remove unnecessary check targetArch !== process.arch

cskeogh committed:
For readability don't escape ', use " instead

Addresses review comment https://github.com/Zettlr/Zettlr/pull/1959#discussion_r637520324
